util/util_test.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRemoveZFromBinds(t *testing.T) {
+	binds := []string{"/etc/kubernetes:/etc/kubernetes:z", "/var/log/kube-audit:/var/log/kube-audit:rw,z", "/var/lib/test:/var/lib/test:Z,ro", "/usr/local/lib/test:/usr/local/lib/test:ro,z,noexec", "/etc/normalz:/etc/normalz"}
+	expectedBinds := []string{"/etc/kubernetes:/etc/kubernetes", "/var/log/kube-audit:/var/log/kube-audit:rw", "/var/lib/test:/var/lib/test:ro", "/usr/local/lib/test:/usr/local/lib/test:ro,noexec", "/etc/normalz:/etc/normalz"}
```

Comment:
Is it worth testing empty binds as well and asserting for failure?

Reply:
which case are we talking about here? `[]string{}` through the function and then it should the same after? or an empty string element?

Reply:
Apologies, I'll be precise: I'm curious what happens if we feed `RemoveZFromBinds` an empty string slice and if we should perform an assertion ("good" or "bad") based on that.

Reply:
done